baselines/test_ad_basics__test_ad_precision_1.dat

```diff
@@ -0,0 +1,4 @@
+codegen_accessor_statements: 36.00
```

Comment:
Are these the number of statements? Can we print them as integers?

Reply:
Thank for pointing out, yes, we can print then as int, but store them in float was not my decision.

Reply:
i guess it's easier for calculating percentages using floats? @xumingkuan

Reply:
Floats are just used for all statistics here.

https://github.com/taichi-dev/taichi/blob/ab2f5dbbf6241b8870ec140a32d4a4e98e78c407/taichi/util/statistics.h#L8

Reply:
Considering that we may need to support float-point numbers in the `Statistics` class, I simply used `float64` when designing this class...